packages/common/src/http.ts

```diff
@@ -1,34 +0,0 @@
-/**
- * A collection of functions to make working with http easier
- * @module
- */
-
-import fs from 'fs';
-import got from 'got';
-
-/**
- * Download a file to disk
- *
- * @param {string} uri - the URI to request
- * @param {string} destination - Where to store file locally
- * @param {Object} [options] - additional download options
- * @param {Object} [options.headers] - headers to include in the request
- * @returns {Promise<undefined>} - resolves when the download is complete
- */
-export const download = (
```

Comment:
I love deleting code, but this wasn't deprecated? I'm not necessarily opposed to removing it regardless, but what led you to do so?

Reply:
I took it out because it was only used in one place, and the replacement code (which I copied from the `got` documentation) was even simpler than this was.